client/src/components/EventCard.js

```diff
@@ -53,9 +63,9 @@ export default function EventCard({ title, imgSrc, date }) {
   return (
     <Card imgSrc={imgSrc}>
       <DateContainer>
-        <div>{days[date.getDay()]}</div>
-        <div>{date.getDate()}</div>
-        <div>{months[date.getMonth()]}</div>
+        <div className="Day">{days[date.getDay()]}</div>
+        <div className="Date">{date.getDate()}</div>
+        <div className="Month">{months[date.getMonth()]}</div>
```

Comment:
What are you using these class names for? Can you not also use styled components for this?

Reply:
Thx for your feedback, I modified this part according to styled components